test_settings.py

```diff
@@ -14,6 +14,7 @@
 }
 
 INSTALLED_APPS = (
+    'django.contrib.admin',
```

Comment:
Q: Why was this additional library needed?

Reply:
I believe this was implicitly added to `INSTALLED_APPS` before Django 2.1 and then needed to be explicitly added after that. (aka, tests failed and this is what the internet told me what I needed to do to fix it.)